integration/busybox.py

```diff
@@ -80,7 +80,9 @@ def create_tarball(self, split=False):
         for path in busybox.stdout.read().split("\n"):
             if not path.strip():
                 continue
-
+            if path.strip() == 'bin/busybox':
```

Comment:
strip the path at the beginning as it get stripped multiple times (below as well)?